.github/workflows/LinuxRelease.yml

```diff
@@ -188,6 +187,48 @@ jobs:
         path: |
           build/release/extension/*/*.duckdb_extension
 
+ linux-extensions-64-aarch64:
+    name: Linux Extensions (AARCH64)
+    runs-on: ubuntu-latest
+    container: ubuntu:18.04 # cross compiler not available in 16
+    needs: linux-release-64
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/ubuntu_16_setup
+        with:
+          openssl: 1
+          aarch64_cross_compile: 1
```

Comment:
Is there a reason we use numbers instead of booleans everywhere for these flags?

Reply:
Good point, they should definitely be booleans, will change this in a next PR